ADA/databricks_rstudio.qmd

```diff
@@ -0,0 +1,70 @@
+---
+  title: "Setup Databricks with RStudio"
+---
+
+<p class="text-muted">
+
+The following instructions set up an ODBC connection between your laptop and your DataBricks cluster, which can then be used in R/RStudio to query data using an ODBC based package.
+
+</p>
+
+------------------------------------------------------------------------
+
+# Pre-requisites
+
+------------------------------------------------------------------------
+
+You must have:
+
+-   Access to Databricks
+-   Access to a SQL Warehouse on DataBricks
+-   R and RStudio downloaded
+
+------------------------------------------------------------------------
+
+# Downloading an ODBC driver
+
+------------------------------------------------------------------------
+
+1.  Install the 'Simba Spark ODBC' driver from the software centre.
```

Comment:
Is there a way to space this out a little more, feels a bit cramped on the page at points? Maybe a break of whitespace after each step?
![image](https://github.com/dfe-analytical-services/analysts-guide/assets/52536248/b6286b4f-7b25-4b39-9d42-9a143a6b1a59)